src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp

```diff
@@ -242,7 +242,7 @@ bool PreFlightCheck::ekf2CheckSensorBias(orb_advert_t *mavlink_log_pub, const bo
 
 		// check accelerometer bias estimates
 		if (bias.accel_bias_valid) {
-			const float ekf_ab_test_limit = 0.5f * bias.accel_bias_limit;
+			const float ekf_ab_test_limit = 0.75f * bias.accel_bias_limit;
```

Comment:
I'd be fine with this change, but what do you think @bresch?

Reply:
With default values this is an accel bias of 0.3 m/s/s.

Reply:
I'm fine with increasing that check unless we start to see issues due to large incorrect accel bias estimate on takeoff.